src/main/java/calculator/util/InputValidator.java

```diff
@@ -0,0 +1,32 @@
+package calculator.util;
+
+import calculator.constants.InvalidInputConstants;
+import java.util.regex.Matcher;
+
+public class InputValidator {
+
+    public static Long validateOnlyPlainNumber(String text) {
+        try {
+            return Long.valueOf(text);
+        } catch (NumberFormatException numberFormatException) {
+            throw new IllegalArgumentException(InvalidInputConstants.WRONG_DELIMITER);
+        }
+    }
+
+    public static void validateCustomFormat(Matcher matcher) {
+        if (!canFindRegex(matcher)) {
+            throw new IllegalArgumentException(InvalidInputConstants.INVALID_CUSTOM_FORMAT);
+        }
+    }
+
+    private static boolean canFindRegex(Matcher matcher) {
+        return matcher.find();
+    }
```

Comment:
부정문을 쓰는거보다 긍정문으로 하는게 좋다고 해요 

```suggestion
    public static void validateCustomFormat(Matcher matcher) {
        if (cannotFindRegex(matcher)) {
            throw new IllegalArgumentException(InvalidInputConstants.INVALID_CUSTOM_FORMAT);
        }
    }

    private static boolean canFindRegex(Matcher matcher) {
        return !matcher.find();
    }
```

이렇게 해보시는건 어떨까요 ??

Reply:
그런 방법이 있었네요! 조금 더 읽기 편한 것 같습니다